2.7.pylintrc

```diff
@@ -0,0 +1,25 @@
+
+[MESSAGES CONTROL]
+
+disable=bad-option-value,misplaced-comparison-constant,no-self-use,consider-iterating-dictionary,
+    consider-using-dict-comprehension,consider-using-set-comprehension,broad-except,
+    useless-object-inheritance,empty-docstring,missing-docstring,missing-module-docstring,
+    missing-class-docstring,missing-function-docstring,line-too-long,trailing-whitespace,
+    missing-final-newline,trailing-newlines,multiple-statements,superfluous-parens,
+    import-outside-toplevel,too-many-statements,bad-whitespace,
+    old-style-class, deprecated-lambda,
+    duplicate-code #TODO:
+
+[SIMILARITIES]
```

Comment:
what does this section do?

Reply:
This are fine tuning options for the `duplicate-code` error message. It determines under which conditions that error will be reported.

Reply:
thanks, may be a good idea to add a comment that this is for duplicate-code

Reply:
Would it be better to just link to the pylintrc options documentation? That would have the description of all of the sections, I think.

Reply:
if you want, you can do that. 

i read the comments below, e.g. "# Minimum lines number of a similarity." and I had no idea what that was. A short comment on the section, e.g. "Settings for duplicate-code" would allow the reader to get a quick idea of what this is about without having to reach out to the documentation.